### PR TITLE
No rounding in compliance chart

### LINKED ISSUE #4642: Improve rounding so we do not round up to 100%
**Is your feature request related to a problem? Please describe.**
Heimdall will report 100% compliance when it is between 99.5% and 99.9%

**Describe the solution you'd like**
Rounding up to 100% should not occur and is misleading that there are 0 failures.

**Describe alternatives you've considered**
None

**Desired Impact to Workflow**
None

image

### PR DESCRIPTION
**Pull request by Amndeep7**
Resolves #4642

Instead of rounding, it does truncation at the end of the math process. Enforces a minimum resolution of four decimal points.

### CODE REVIEW

**Comment by ejaronne** [comment]
As long as it truncates, I think 2 decimal points should be fine.

**Review by ejaronne — DISMISSED** [review]
As long as it truncates, I think 2 decimal points is fine. Looks a little silly at 4 decimal points.

**Review by em-c-rod — APPROVED** [review]

File: apps/frontend/src/components/cards/ComplianceChart.vue [hunk 1]
```diff
@@ -45,7 +47,12 @@ export default class ComplianceChart extends Vue {
             show: true,
             value: {
               color: '#99a2ac',
-              fontSize: '2rem'
+              fontSize: '2rem',
+              formatter: (val: number) =>
+                `${(
+                  Math.trunc(Math.pow(10, MAX_DECIMAL_PRECISION) * val) /
```
Comment (Hookwitz) [inline apps/frontend/src/components/cards/ComplianceChart.vue:53]: Mathematically speaking, both `Math.pow()` functions cancel each other out. So neither is really necessary and could be rewritten to just `Math.trucn(val).toFixed(MAX_DECIMAL_PRECISION)`, right?

**Review by Hookwitz — CHANGES_REQUESTED** [review]

File: apps/frontend/src/components/cards/ComplianceChart.vue (hunk 1 above)
Comment (Amndeep7) [reply apps/frontend/src/components/cards/ComplianceChart.vue:53]: nope cause math.trunc cuts of all decimal values.  consequently i bring as much decimal precision as i need over to the left of the decimal point, chop it all off with trunc, and then shove it back over the dot again with the divide

**Pull request merged**